[2025-06-19T12:16:54.472+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: week_3_weather_api_dag.validate_data manual__2025-06-19T12:16:49.824208+00:00 [queued]>
[2025-06-19T12:16:54.483+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: week_3_weather_api_dag.validate_data manual__2025-06-19T12:16:49.824208+00:00 [queued]>
[2025-06-19T12:16:54.483+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 1
[2025-06-19T12:16:54.500+0000] {taskinstance.py:2192} INFO - Executing <Task(PythonOperator): validate_data> on 2025-06-19 12:16:49.824208+00:00
[2025-06-19T12:16:54.509+0000] {standard_task_runner.py:60} INFO - Started process 90 to run task
[2025-06-19T12:16:54.520+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'week_3_weather_api_dag', 'validate_data', 'manual__2025-06-19T12:16:49.824208+00:00', '--job-id', '43', '--raw', '--subdir', 'DAGS_FOLDER/week_3_weather_api_dag.py', '--cfg-path', '/tmp/tmpbem5i8ko']
[2025-06-19T12:16:54.521+0000] {standard_task_runner.py:88} INFO - Job 43: Subtask validate_data
[2025-06-19T12:16:54.592+0000] {task_command.py:423} INFO - Running <TaskInstance: week_3_weather_api_dag.validate_data manual__2025-06-19T12:16:49.824208+00:00 [running]> on host b96384fab38e
[2025-06-19T12:16:54.684+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='week_3_weather_api_dag' AIRFLOW_CTX_TASK_ID='validate_data' AIRFLOW_CTX_EXECUTION_DATE='2025-06-19T12:16:49.824208+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-19T12:16:49.824208+00:00'
[2025-06-19T12:16:54.686+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-06-19T12:16:54.700+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=week_3_weather_api_dag, task_id=validate_data, execution_date=20250619T121649, start_date=20250619T121654, end_date=20250619T121654
[2025-06-19T12:16:54.736+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-06-19T12:16:54.767+0000] {taskinstance.py:3281} INFO - 1 downstream tasks scheduled from follow-on schedule check
